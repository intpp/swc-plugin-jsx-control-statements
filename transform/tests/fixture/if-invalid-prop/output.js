var React = require("react");
module.exports = class extends React.Component {
    render() {
        return <div>
                {false ? null : null}
            </div>;
    }
};
