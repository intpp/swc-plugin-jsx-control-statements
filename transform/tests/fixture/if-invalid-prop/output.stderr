  x Unsupported: "blablabla" for <If>, valid props: "condition" and "key"!
   ,-[input.js:7:1]
 6 |             <div>
 7 |                 <If blablabla={true}>
   :                     ^^^^^^^^^
 8 |                 </If>
   `----
  x Attribute "condition" is required for the <If> tag!
   ,-[input.js:7:1]
 6 |             <div>
 7 |                 <If blablabla={true}>
   :                 ^^^^^^^^^^^^^^^^^^^^^
 8 |                 </If>
   `----
